docs/guides-publish.md

```diff
@@ -144,13 +140,13 @@ We will fetch the published versions of the official `voting` app on the rinkeby
 Command:
 
 ```sh
-npx aragon apm voting.aragonpm.eth --environment rinkeby
+aragon apm versions voting.aragonpm.eth --environment rinkeby
 ```
 
 Sample output:
 
 ```sh
-ℹ voting.aragonpm.eth has 13 published versions
+ℹ voting.aragonpm.eth has 24 published versions
```

Comment:
I think there's only 13 listed here

Reply:
Yes I only printed about half of the versions so it could take less space, but the command prints them all.